dev/TEAL_opcodes.md

```diff
@@ -24,23 +24,23 @@ Ops have a 'cost' of 1 unless otherwise specified.
 - Pops: *... stack*, []byte
 - Pushes: []byte
 - SHA256 hash of value X, yields [32]byte
-- **Cost**: 7
+- **Cost**: 35
```

Comment:
We need a way to say that at version 1 it was 7 and at version 2 it is 35 (and also make that true in Check/Eval)

Reply:
done